cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/k8s/PreviewRunnerMain.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.app.runtime.k8s;
+
+import com.google.common.util.concurrent.Service;
+import com.google.inject.Injector;
+import com.google.inject.Module;
+import io.cdap.cdap.common.logging.LoggingContext;
+import io.cdap.cdap.master.environment.k8s.AbstractServiceMain;
+import io.cdap.cdap.master.environment.k8s.EnvironmentOptions;
+import io.cdap.cdap.master.spi.environment.MasterEnvironment;
+import io.cdap.cdap.master.spi.environment.MasterEnvironmentContext;
+
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * Run a preview.
+ */
+public class PreviewRunnerMain extends AbstractServiceMain<EnvironmentOptions> {
```

Comment:
This is not used in the current PR, however will be used by subsequent PR where StatefulSet impl will be added.

Reply:
Can you remove this and include it back in the next PR? Ideally all code merged should be production ready.

Reply:
sure. removed.